part1/chapter6/quiz1/questions/question_2.rb

```diff
@@ -20,3 +20,22 @@
 # * Note: You can assume the players will only ever input `rock`,
 #   `paper` or `scissors`.
 
+puts "This is a two player game \"Rock, Paper, Scissors\", you can only input either "rock", "paper" or "scissors" "
+puts "Player 1 do your move: "
+
+
+p1_input = gets.chomp
+#total_p1_input << p1_input
+
+puts "Player 2 do your move: "
+p2_input = gets.chomp
+
+  if (p1_input == "rock" && p2_input == "rock" )|| (p1_input == "paper" && p2_input == "paper") || (p1_input == "scissors" && p2_input == "scissors")
```

Comment:
It's good practice for the indentation of the `end` to match with the indentation of `if`